src/modules/verifiableCredentials/verifiableCredentialsBase.service.ts

```diff
@@ -131,6 +159,125 @@ export abstract class VerifiableCredentialsServiceBase {
       signature
     );
 
-    return JSON.parse(signedCredential) as SignedVerifiableCredential<T>;
+    return JSON.parse(
+      signedCredential
+    ) as VerifiableCredential<EWFCredentialSubject>;
+  }
+
+  public createPresentation(
```

Comment:
Speaking of PEX, instead of this function, could we use `pex.presentationFrom` instead? https://github.com/Sphereon-Opensource/pex#holder-presentation-creation-non-verifiable

Reply:
Definitely! I will try to use as many features from PEX as I can

Reply:
Problem is that this methods are doing different things. Method that I implemented is creating from a given VCs, but PEX require VCs and presentation definition. Do we want to replace my method with that from PEX or maybe we can create another method

Reply:
Ah, I see, the problem is that PEX method requires a `presentationDefinition`. Hmm. I think we should definitely use the pex method as . However, I think it would be code to encapsulate it so that we can change the implementation or support other presentation types. At the moment, my suggestion is an API like this:
```
createPresentation(verifiableCredentials: VerifiableCredential<EWFCredentialSubject>[], options: createPresentationOptions)
```
Where `createPresentationOptions` looks like this:
```
{
  type: PresentationType, // e.g. "PresentationExchangeSubmission"
  presentationDefinition?: PresentationDefinition
}
```
Or maybe it's simpler just to do
```
createPEXPresentation(verifiableCredentials: VerifiableCredential<EWFCredentialSubject>[], presentationDefinition: PresentationDefinition)
```
What do you think?